gto/config.py

```diff
@@ -27,45 +33,47 @@ def load(self) -> EnrichmentReader:
 
 class NoFileConfig(BaseSettings):  # type: ignore[valid-type]
     INDEX: str = "artifacts.yaml"
-    TYPES: Optional[List[str]] = None
-    STAGES: Optional[List[str]] = None
+    CONFIG_FILE_NAME: Optional[str] = CONFIG_FILE_NAME
     LOG_LEVEL: str = "INFO"
     DEBUG: bool = False
-    ENRICHMENTS: List[EnrichmentConfig] = []
-    AUTOLOAD_ENRICHMENTS: bool = True
-    CONFIG_FILE_NAME: Optional[str] = CONFIG_FILE_NAME
     EMOJIS: bool = True
 
-    class Config:
-        env_prefix = "gto_"
+    types: Optional[List[str]] = None
+    stages: Optional[List[str]] = None
+    enrichments: List[EnrichmentConfig] = Field(default_factory=list)
+    autoload_enrichments: bool = True
```

Comment:
In pydantic v1, these fields were case-insensitive and converted to lowercase fields.

That is no longer the case.